spring-cloud-gcp-autoconfigure/pom.xml

```diff
@@ -176,6 +176,11 @@
             <optional>true</optional>
         </dependency>
 
+        <dependency>
+            <groupId>com.google.api.grpc</groupId>
+            <artifactId>grpc-google-cloud-firestore-v1</artifactId>
+            <optional>true</optional>
```

Comment:
This is not brought in transitively from `com.google.cloud:google-cloud-firestore`?

Reply:
Good catch!